New log file opened at Wed Feb 01 14:02:48 COT 2017 by 8928@DESKTOP-EUGFD9N
Taking next connection to: 63149
New client attempting to connect on port: 63150-63149
  Address = /192.168.0.2:63149
This client on port 63150 has uuid 5d8a6a1e-ca60-4f3f-9d1b-1ab7d31a4898 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-5d8a6a1e-ca60-4f3f-9d1b-1ab7d31a4898
Taking next connection to: 63149
New client attempting to connect on port: 63151-63149
  Address = /192.168.0.2:63149
Handshake exception on socket: 63151
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 63149
New client attempting to connect on port: 63152-63149
  Address = /192.168.0.2:63149
This client on port 63152 has uuid 14783f7f-24d5-4950-9fae-b0345e267ff7 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-5d8a6a1e-ca60-4f3f-9d1b-1ab7d31a4898, activator-14783f7f-24d5-4950-9fae-b0345e267ff7
Taking next connection to: 63149
New client attempting to connect on port: 63158-63149
  Address = /192.168.0.2:63149
Handshake exception on socket: 63158
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 63149
New client attempting to connect on port: 63159-63149
  Address = /192.168.0.2:63149
This client on port 63159 has uuid 7cdef407-77bd-4ece-af48-53d38a76514b configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-14783f7f-24d5-4950-9fae-b0345e267ff7, play-fork-run-7cdef407-77bd-4ece-af48-53d38a76514b
Taking next connection to: 63149
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 63149
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 63149
Checking to see if clients are empty...
No non-daemon clients connected after 3 min. Shutting down.
1 daemon clients are connected.
Server socket thread exiting.
1 clients open, will close them...
All client sockets have been closed.
Closing the 8928@DESKTOP-EUGFD9N logs at Wed Feb 01 14:12:31 COT 2017, goodbye.
